sdks/python/apache_beam/transforms/timeutil.py

```diff
@@ -115,8 +115,8 @@ def __mod__(self, other):
     return Duration(micros=self.micros % other.micros)
 
 
-MIN_TIMESTAMP = Timestamp(micros=-sys.maxint - 1)
-MAX_TIMESTAMP = Timestamp(micros=sys.maxint)
+MIN_TIMESTAMP = Timestamp(micros=-0x7fffffffffffffff - 1)
```

Comment:
This would work but would likely have a performance cost.  In Python, when integers overflow, bigints (Python "long"s) are transparently used, since 0x8000000000000000 is out of range for a 64-bit signed integer.  This has a performance cost, since then the negative of this, -0x8000000000000000, is still treated as a bigint and won't be converted back to a regular 64-bit integer.